### PR TITLE
Hunter Stealth Attack Nerf

### PR DESCRIPTION
**Pull request by InterroLouis**
## About The Pull Request
Basically it reduces the damage multiplier max for the hunter stealth attack. 

## Why It's Good For The Game
Less stealth one hit kills/delimbs. Hopefully. 
## Changelog
:cl:
balance: Hunter stealth attack damage multiplier has been reduced.
/:cl:

### CODE REVIEW

**Comment by Spyroshark** [comment]
Have you considered improving other parts of the caste? Because right now the only thing hunter is good at is stealth attacks, other then that it's just a worse runner.

**Comment by spookydonut** [comment]
> 
> 
> Have you considered improving other parts of the caste? Because right now the only thing hunter is good at is stealth attacks, other then that it's just a worse runner.

basically this.

**Comment by Rohesie** [comment]
Same consideration.
Hunter stealth attack is an issue, that can't be denied. We in general we can't just remove every issue, else we'd be left with no game.
The Hunter needs something to make it appealing. Even if that means the old punce-stun.

**Comment by InterroLouis** [comment]
This isn't even removing it-it's a rebalance. This has been an issue for a while, so I'm addressing it with this PR. I'm not going to pretend to be a good enough coder to add in more things to make the hunter more appealing, but I will say that the amount of damage it caused with the stealth attack was frankly ridiculous. If someone else wants to change things for that, they can feel free to, just like I'm looking to change this.

File: code/modules/mob/living/carbon/xenomorph/castes/hunter/hunter.dm [hunk 1]
```diff
@@ -74,7 +74,7 @@
 		alpha = HUNTER_STEALTH_WALK_ALPHA //80% invisible
 		use_plasma(HUNTER_STEALTH_WALK_PLASMADRAIN * 0.5)
 		if(sneak_bonus < HUNTER_SNEAKATTACK_MAX_MULTIPLIER)
-			sneak_bonus = round(min(sneak_bonus + HUNTER_SNEAKATTACK_WALK_INCREASE, 3.5), 0.01) //Recover sneak attack multiplier rapidly
+			sneak_bonus = round(min(sneak_bonus + HUNTER_SNEAKATTACK_WALK_INCREASE, 2.0), 0.01) //Recover sneak attack multiplier rapidly
```
Comment (spookydonut) [inline code/modules/mob/living/carbon/xenomorph/castes/hunter/hunter.dm]:
```suggestion
			sneak_bonus = round(min(sneak_bonus + HUNTER_SNEAKATTACK_WALK_INCREASE, HUNTER_SNEAKATTACK_MAX_MULTIPLIER), 0.01) //Recover sneak attack multiplier rapidly
```

**Pull request merged**